src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IObjectCreationExpression.cs

```diff
@@ -3423,30 +3423,36 @@ public void M(bool b)
     Locals: [C1 x]
     Block[B1] - Block
         Predecessors: [B0]
-        Statements (1)
+        Statements (2)
             IFlowCaptureOperation: 0 (OperationKind.FlowCapture, Type: null, IsInvalid, IsImplicit) (Syntax: 'new C1 { O[ ... 2] = null }')
               Value: 
                 IObjectCreationOperation (Constructor: C1..ctor()) (OperationKind.ObjectCreation, Type: C1, IsInvalid) (Syntax: 'new C1 { O[ ... 2] = null }')
                   Arguments(0)
                   Initializer: 
                     null
 
+            IFlowCaptureOperation: 1 (OperationKind.FlowCapture, Type: null, IsInvalid, IsImplicit) (Syntax: 'O')
```

Comment:
>IFlowCaptureOperation: 1 (OperationKind.FlowCapture, Type: null, IsInvalid, IsImplicit) (Syntax: 'O') [](start = 12, length = 101)

This change in evaluation order feels unexpected. Could you please sync-up with @333fred and confirm order of evaluation in IL.